src/server/routes/monthData.js

```diff
@@ -68,16 +68,35 @@ function currentMonthData(req, res, next) {
     });
 }
 
+/**
+ * Returns first two matches hyphenated for redirecting a three part date
+ * to a two part date in a URL.
+ * @param {object} match
+ * @param {integer} d1
+ * @param {integer} d2
+ * @return {string}
+ */
+function toMonthAndYear(match, d1, d2) {
```

Comment:
The argument `match` doesn't do anything in the function. We can remove it?

Reply:
Sorry I was wrong after reading more into the document. Please ignore the comment.

Reply:
It's a defined param for the callback function, but it is weird that it's not being used in this function itself. Removing it may break the functional call, even if it's not being used.

Reply:
@gkallenberg this could go in the `utils`.